src/cl65/main.c

```diff
@@ -156,7 +156,21 @@ static char* TargetLib  = 0;
 #  endif
 #endif
 
+/*****************************************************************************/
+/*                        Credential functions                               */
+/*****************************************************************************/
+void DisableAssembling(void){
```

Comment:
In general please adhere to the predominant style:
- blank before parameter list
- open brace for function boy on own line
- 'static' if not referenced from other module